### DISCUSSION TITLE
About "Cargo didn't build a cdylib..."

### DISCUSSION THREAD

**Discussion by km19809** [PyO3/maturin#1140] [Q&A] [answered]
After the build command, maturin fails with a message: `Cargo didn't build a cdylib. Did you miss crate-type = ["cdylib"] in the lib section of your Cargo.toml?`
However, the crate was compiled successfully.
Also, there is crate-type = ["cdylib"] in the lib section. [here](https://github.com/km19809/pyxel/blob/d8601a3b13071a605d48ec9a8071ef509f75bb1d/crates/pyxel-extension/Cargo.toml#L15).

What happened to me?
How can I solve this problem?

The last 4 lines of the message:
```
   Compiling unindent v0.1.10
    Finished release [optimized] target(s) in 1m 12s
💥 maturin failed
  Caused by: Cargo didn't build a cdylib. Did you miss crate-type = ["cdylib"] in the lib section of your Cargo.toml?
```

**Environments**
Rust: 1.64.0
Rust-nightly: 1.66.0-nightly (8b705839c 2022-09-26)
Host: x86_64-unknown-linux-gnu

Maturin: 0.13.4

**How to reproduce**
1) Clone a repository. 
`git clone https://github.com/km19809/pyxel`
2) Build with maturin.
`RUSTUP_TOOLCHAIN=nightly maturin build -o dist --release --target wasm32-unknown-emscripten -- -Clink-arg=-sUSE_SDL=2`

**Comment by messense** [comment, marked as answer]
Thanks for the report, I think it's a regression in maturin 0.13.4.

Should be fixed by https://github.com/PyO3/maturin/pull/1142
